src/App.tsx

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
 import './App.scss';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './Components/TodoList';
 
-// import todos from './api/todos';
-// import users from './api/users';
+const preparedTodos = todos.map((todo) => ({
+  ...todo,
+  user: users.find((u) => u.id === todo.userId),
```

Comment:
If you define in interface Todo that user can be null, you should add it in code:
```suggestion
  user: users.find((u) => u.id === todo.userId) || null,
```